mp/src/game/client/view_effects.cpp

```diff
@@ -394,7 +375,7 @@ screenshake_t *CViewEffects::FindLongestShake()
 //-----------------------------------------------------------------------------
 void CViewEffects::Shake( ScreenShake_t &data )
 {
-	if ( ( data.command == SHAKE_START || data.command == SHAKE_START_RUMBLEONLY ) && ( m_ShakeList.Count() < MAX_SHAKES ) )
+	if ( ( data.command == SHAKE_START ) && ( m_ShakeList.Count() < MAX_SHAKES ) )
```

Comment:
Huh, I wonder if this was a bug on valve's part, shouldn't that second check be `== SHAKE_START_NORUMBLE`?

Reply:
I was thinking that too, actually, should I change that?

Reply:
Yeah I would make it check both, this can be called with start and with start_norumble